src/containers/BooksForm.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+const BooksForm = ({ dispatch }) => {
+  const [state, setState] = useState({
+    id: Date.now(),
+    title: '',
+    category: 'Action',
+  });
+  const handleChange = (e) => {
+    if (e.target.name === 'title') {
+      setState({
+        ...state, title: e.target.value,
+      });
+    }
+    if (e.target.name === 'category') {
+      setState({
+        ...state, title: e.target.value,
+      });
+    }
+  };
```

Comment:
[**OPTIONAL**]

It would be nice if you refactor this code like so:

``` 
handleChange({ target: { name, value } }) {
  this.setState({
    [name]: value,
  })
}

```